src/routes/auth/user/components/Main.svelte

```diff
@@ -0,0 +1,89 @@
+<script>
+  import Step from './Step.svelte';
+  import Footer from './Footer.svelte';
+  export let repos;
+  export let users;
+  console.log('repoArray', repos);
+  console.log('usersArray', users);
+  console.log('repoArray', repos[0].html_url);
```

Comment:
I am not sure if `vite` removes console logs in the production build, but it's better not to bloat code with them